llvm/include/llvm/Transforms/Vectorize/SandboxVectorizer/DependencyGraph.h

```diff
@@ -159,8 +168,37 @@ class DependencyGraph {
   /// The DAG spans across all instructions in this interval.
   Interval<Instruction> DAGInterval;
 
+  std::unique_ptr<BatchAAResults> BatchAA;
+
+  enum class DependencyType {
+    RAW,   ///> Read After Write
+    WAW,   ///> Write After Write
+    RAR,   ///> Read After Read
+    WAR,   ///> Write After Read
+    CTRL,  ///> Dependencies related to PHIs or Terminators
```

Comment:
what does `CRTL` stand for?

Reply:
Ah it stands for CONTROL, I will update the comment.